domain.yml

```diff
@@ -1,753 +1,860 @@
-  utter_ask_feedback:
-    - text: "How is this conversation going?"
-      buttons:
-        - title: "👍"
-          payload: "/feedback{\"feedback_value\":\"positive\"}"
-        - title: "👎"
-          payload: "/feedback{\"feedback_value\":\"negative\"}"
+  - text: Goodbye!
+  - text: Bye!
+  - text: Ciao!
+  - text: Bye, was nice talking to you!
+  utter_can_do:
+  - text: Sure thing!
+  - text: Can do!
+  - text: "You got it. 👍"
+  utter_cant_answer_tech_question:
+  - text: I can't answer that.
   utter_canthelp:
-    - text: "I'm sorry I couldn't help you. If you have general questions about the Stack, please head to https://forum.rasa.com/, otherwise contact us at hi@rasa.com for anything else."
-  utter_nohelp:
-    - text: "For now that's all I can do, so feel free to try it out. If you have any suggestions for what I should learn, let me know!"
-  utter_thumbsup:
-    - text: "👍"
-    - text: "Okay."
-    - text: "Got it."
+  - text: I'm sorry I couldn't help you. If you have general questions about the Rasa,
+      please head to https://forum.rasa.com/, otherwise contact us at hi@rasa.com for
+      anything else.
   utter_cantsignup:
-    - text: "I can't sign you up if you don't provide your email 😉"
-  utter_ask_isbot:
-    - text: "Yep, I'm a bot!"
-  utter_cant_answer_tech_question:
-    - text: "I can't answer that."
-  utter_no_email:
-    - text: "Hmm, I'm not sure that's a valid email, please make sure to include the full address 😅"
-  utter_have_you_used_rasa_before:
-    - text: "Have you used Rasa before?"
-  utter_built_bot_before:
-    - text: "Alright, have you built a bot before?"
-  utter_anything_else:
-    - text: "Is there anything else I can help you with?"
-  utter_encourage_building_bot:
-    - text: Cool, then good luck with building your first "I'm the Rasa bot 🤖!
+  - text: "I can't sign you up if you don't provide your email 😉"
+  utter_change_mind:
+  - text: You can still change your mind below though.
   utter_chatbot_tutorial:
-    - text: "No worries. There's a course on Datacamp that teaches you how to build your own chatbots: https://www.datacamp.com/courses/building-chatbots-in-python 🤓"
-  utter_no_guide_for_switch:
-    - text: "Sorry, but we don't have a guide for this yet."
-  utter_ask_which_product:
-    - text: "Ok, would you like to know more about Core or NLU?"
+  - text: "No worries. There's a course on Datacamp that teaches you how to buildyour\
+      \ own chatbots: https://www.datacamp.com/courses/building-chatbots-in-python \
+      \ 🤓"
+  utter_confirm_salesrequest:
+  - text: "Thanks a bunch, we'll be in touch soon! ✉️"
+  utter_confirmationemail:
+  - text: We sent a confirmation email to {email}.
+  utter_contact_email:
+  - text: "You can't speak to a human directly in this chat, but you can email\
+      \ us at hi@rasa.com."
+  utter_continue_step2:
+  - buttons:
+    - payload: /get_started_step2
+      title: Continue
+    text: Let's continue, click the button below
```

Comment:
@msamogh did you take a look at the utterances on `website-demo.rasa.com` to define channel specific templates here? Because like i said these are different for hte socketio channel

Reply:
Not sure what you mean by "channel-specific templates" @akelad

Reply:
did you take a look at the utterances on the instance i'm talking about? They're different to what is defined here, because we do some redirecting to pages when the user clicks a button. 
This is what i mean: https://rasa.com/docs/rasa/core/domains/#channel-specific-utterances